alignak/scheduler.py

```diff
@@ -1774,16 +1782,20 @@ def get_new_broks(self):
             elt.broks = []
 
     def check_freshness(self):
-        """Iter over all hosts and services to check freshness
+        """
+        Iter over all hosts and services to check freshness if check_freshness enabled and
+        passive_checks_enabled enabled
 
         :return: None
         """
-        # print "********** Check freshness******"
         for elt in self.iter_hosts_and_services():
-            chk = elt.do_check_freshness(self.hosts, self.services, self.timeperiods,
-                                         self.macromodulations, self.checkmodulations, self.checks)
-            if chk is not None:
-                self.add(chk)
+            if elt.check_freshness and elt.passive_checks_enabled:
```

Comment:
what are you keeping? Stack write? We are doing double check here. If we want to change the behavior we will need to this it in both place